CONTRIBUTING.md

```diff
@@ -1,4 +1,43 @@
-# Contributing
+# Contributing to BosqueLanguage
+
+We love your input! We want to make contributing to this project as easy as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+
+# How Do I Submit A (Good) Bug Report?
+
+Explain the problem and include additional details to help maintainers reproduce the problem:
```

Comment:
Let's include "Check if the same, or a similar, issue has already been reported" to minimize duplicates and help connect issues that may be related.